infrastructure/cluster/flux-v2/jhub/jhub-ns.yaml

```diff
@@ -0,0 +1,6 @@
+# kind: Namespace
```

Comment:
Because it first needs to be removed from Terraform to avoid the whole ns getting deleted.

Reply:
Then which is the correct way to follow ? (in case I find a similar situation in the future). 
Prepare flux, remove the ns from tf and then sync it with flux ?

Reply:
Actually, it's a bit more tricky. You don't want to destroy the tf namespace resource but rather remove the code and then [clean the tf state](https://developer.hashicorp.com/terraform/cli/commands/state/rm). Then add the exact same namespace yaml to flux, to avoid it getting deleted with everything inside!

Reply:
While cleaning tf state, I noticed the reana stuff is also still in there, but now also created with flux @egazzarr ? I'll remove it from the state. I guess jhub is the only piece missing, so we can fully clean tf state tomorrow?

Reply:
yes you can delete the tf completely as concerns reana!

Reply:
Except for the share? But let's talk tomorrow then.